salt/serializers/configparser.py

```diff
@@ -21,18 +21,37 @@
 available = True
 
 
+def _check_preservecase_option(**options):
+    '''
+    Checks options passed to the serialize/deserialze function
+    for the 'casesensitive' argument.  If passed as True,
+    the configparser object will preserve case in items
+    '''
+    preserve_case = False
+    if 'preserve_case' in options:
+        if options['preserve_case']:
+            preserve_case = True
+        options.pop('preserve_case')
```

Comment:
actually you can replace the 5 lines with the preserve_case = options.pop('preserve_case', False)